FlightReservation/App.js

```diff
@@ -15,6 +17,7 @@ const App = () => {
       <Provider store={store}>
         <Stack.Navigator screenOptions={{header: () => null}}>
           <Stack.Screen name="SignUp" component={SignUp} />
+          <Stack.Screen name="LogIn" component={LogIn} />
```

Comment:
Recommendation, this could be a map of an array of objects (You will add severals stacks navigators) but in general, is good